cmd/org/aws-accounts.go

```diff
@@ -45,7 +45,7 @@ func init() {
 		"ou-id",
 		"",
 		"",
-		"specify orgnaization unit id",
+		"specify organization unit id",
```

Comment:
I can't remember why I came across this as part of this work... but either way I fixed it in here.